homework/index.js

```diff
@@ -29,8 +29,42 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function renderRepoDetails(repo) {
+    const reposList = document.getElementById('reposList');
+    const repoItem = createAndAppend('li', reposList);
+    const commonBlock = createAndAppend('div', repoItem, {
+      class: 'commonBlock',
```

Comment:
By convention, CSS classed should be in **kebab-case**: `common-block`.  This holds true for a couple of places in your code (classes, ids).

Reply:
Good to know! Thank you